app/scripts/views/components/barChart/barChartTemplate.js

```diff
@@ -186,7 +188,15 @@
         width: this.settings.width,
         height: this.settings.height
       });
-
+      groupNdxData = iViz.getGroupNdx(this.opts.groupid);
+      
+      if(this.opts.attrId === 'mutation_count') {
```

Comment:
I add mutation_count to data_.groups.sample.data[_sampleDatum.sample_uid].mutation_count